services/QuillLMS/app/services/vitally_integration/serialize_vitally_sales_user.rb

```diff
@@ -94,7 +94,13 @@ def data
         completed_evidence_activities_per_student_last_year: get_from_cache("completed_evidence_activities_per_student"),
         date_of_last_completed_evidence_activity: date_of_last_completed_evidence_activity,
         premium_state: @user.premium_state,
-        premium_type: @user.subscription&.account_type
+        premium_type: @user.subscription&.account_type,
+        role: @user.role,
+        admin_sub_role: @user.admin_sub_role,
+        email_verification_status: @user.email_verification_status,
+        admin_approval_status: @user.admin_approval_status,
+        admin_of_schools: admin_of_schools,
```

Comment:
Should we be at all concerned about how long this could be?  If someone is the admin of, say, 100 schools, will this payload get big enough to break something?  And in cases where it doesn't break something, is a list of 100 schools going to be useful to anyone?

Reply:
Yeah, this is a good question. I wasn't able to find anything in either Vitally or Segment's docs about a maximum length, but we do have some users with over 100 schools. I'll ask Peter S if it makes sense to add some kind of cap, or if we'd rather release as is and fix if necessary.

Reply:
That seems reasonable.  The stakes feel like they're probably not too high if we get this wrong since the Segment code is isolated javascript, so if it 500s, I don't think it'll impact user experience.  And the Vitally data is synced overnight in Sidekiq.  So if our payloads get too big even though there aren't documented limits, it should be invisible to users so we can clean it up later.

Reply:
Discussed with Peter S and we decided to turn these values into numbers: https://www.notion.so/Implement-the-necessary-changes-to-Segment-and-Vitally-to-track-admin-creation-and-usage-540e50dfe1694250a4e015c2e566312a?d=ce46559e466d4b78a78ad8287dc81d2e#50364f14078c4b48936f1d150e646dbe.